src/telemetry/telemetry.c

```diff
@@ -16,19 +25,59 @@
 
 #define MODULE_COUNT 2
 
+static downlink_node *downlink_queue = NULL;
+
 /* This list controls the order of initialisation */
 static const module_init_t init_sequence[MODULE_COUNT] = {
-    {"downlink", &init_downlink},
-    {"downlink_queue", &init_downlink_queue}
+        {"downlink",       &init_downlink},
+        {"downlink_queue", &init_downlink_queue}
 };
 
-int init_telemetry( void ){
+// TODO: For test purposes only!
+void *test_telemetry_generator() {
```

Comment:
I think we need to design a system for code testing. Perhaps for now this should be removed to not clog up the production code with testing?

Reply:
Yeah, I'll keep the code itself for now, but I'll comment it out, so it's not clogging the workflow of the daemon.